srcs/back/src/friends/friends.service.ts

```diff
@@ -0,0 +1,71 @@
+import { Injectable, NotFoundException } from '@nestjs/common'
+import { PrismaService } from 'src/prisma/prisma.service'
+import { User } from '@prisma/client'
+import { userInfo } from 'os'
+
+@Injectable()
+export class FriendsService {
+  constructor(private prisma: PrismaService) {}
+
+  async findUser(userId: number) {
+    return this.prisma.user.findUnique({
+      where: {
+        id: userId
+      },
+      include: {
+        friends: true
+      },
+    })
+  }
+
+  async addFriend(userId: number, friendId: number): Promise<User> {
+    return await this.prisma.user.update({
```

Comment:
```suggestion
    return this.prisma.user.update({
```
await non necessaire ici

Reply:
https://stackoverflow.com/a/42750371